Jenkinsfile

```diff
@@ -13,7 +13,7 @@ kind: Pod
 spec:
   containers:
   - name: amlen-centos7-build
-    image: quay.io/amlen/amlen-builder-centos7:1.0.0.5
+    image: quay.io/amlen/amlen-builder-centos7:latest
```

Comment:
Is this change deliberate? I thought we changed to specific versions so when we branch off e.g. 1.1 maintenance branch its not broken by dev changes?

Reply:
I've now changed it to 1.0.0.6 although trying to rebuild the other images all seem to be hanging with no helpful output